### PR TITLE
Car manual control via bluetooth

### PR DESCRIPTION
**Pull request by GeorgeKarageozian**
## Description
Basic maneuvering over the car using pre-downloaded application on a mobile device
## Related issue(s)
#10 #4

### CODE REVIEW

File: ArduinoCar_v2/ArduinoCar_v2.ino [hunk 1]
```diff
@@ -1,28 +1,66 @@
...
+int speedTurn = 38;
+char t ;
 BrushedMotor leftMotor(8, 10, 9);
 BrushedMotor rightMotor(12, 13, 11);
 DifferentialControl control(leftMotor, rightMotor);
 
 SR04 front(TRIGGER_PIN, ECHO_PIN, MAX_DISTANCE);
-
+SoftwareSerial BTSerial(0,1);
 SimpleCar car(control);
```
Comment (Fredrik93) [inline ArduinoCar_v2/ArduinoCar_v2.ino]: we should use smartcar()

**Review by DetailsMatterM — CHANGES_REQUESTED** [review]
I suggest that we move the code that is commented out, so the code for obstacle avoidance, to it's own arduino sketch to avoid bulky sketches

**Review by Fredrik93 — DISMISSED** [review]
It looks great ! 
I think a good implementation would be to have the car stop whenever we input "wrong " letters e.g not w,a,s,d .. and also change the letters to w,a,s,d :)

**Review by GeorgeKarageozian — COMMENTED** [review]
Changes made

**Review by GeorgeKarageozian — COMMENTED** [review]
Changes made

**Review by DetailsMatterM — APPROVED** [review]
Looks great, approved

**Comment by Adamshw** [comment]
Looks great. Agreed on what Mortiz said, other than that everything looks nice.

**Pull request merged**